packages/calcite-components/src/utils/floating-ui.ts

```diff
@@ -508,6 +508,8 @@ export function connectFloatingUI(
     component,
     runAutoUpdate(referenceEl, floatingEl, () => component.reposition()),
   );
+
+  component.reposition(true);
```

Comment:
Won't this cause the component to reposition twice (once via `runAutoUpdate`/`autoUpdate` and again here) when connected?

Reply:
I don't think so because when connected, the reference and floatingEl elements are not yet created.

The reason we need this for both connectedCallback and componentDidLoad is because the elements are not yet created in connectedCallback but if a component is moved within the DOM, we do need to reconnect it. So both should be necessary.

Reply:
If runAutoUpdate does reposition, initially, then we might not need this but we had it setup like this already so I'm hesitant to remove it.

Reply:
@jcfranco do you know if autoUpdate is called initially?

Reply:
[It appears so](https://github.com/floating-ui/floating-ui/blob/e642116c9e3fea62b8e275a8dcb75c9109b19350/packages/dom/src/autoUpdate.ts#L211).